pandas/indexes/api.py

```diff
@@ -0,0 +1,116 @@
+from .base import (Index, _new_Index,  # noqa
+                   _ensure_index, _get_na_value,
+                   InvalidIndexError)
+from .category import CategoricalIndex  # noqa
+from .multi import MultiIndex  # noqa
+from .numeric import NumericIndex, Float64Index, Int64Index  # noqa
```

Comment:
the relative imports are not consisten with the rest of the import methodology. Ok, but should define when relative imports are ok.